src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -1,13 +1,22 @@
 (ns status-im2.contexts.chat.messages.content.text.style
   (:require [quo2.foundations.colors :as colors]))
 
+(def spacing-between-blocks 10)
+
+(def parsed-text-block
+  {:margin-top -4})
```

Comment:
using negative margin looks suspicious

Reply:
why? it's because each paragraph has a margin-top set
and this particular offset is for the whole block itself, to compensate the margin-top offset for the first paragraph